content/docs/guide-development.md

```diff
@@ -117,6 +117,34 @@ https://renkulab.io/
 
 TODO: explain how to run server
 
+### Developing plugins for Renku
```

Comment:
I do not this this section is needed here like that, the workflow developers will not develop renku plugins.

Reply:
Yes, true. I just wanted to introduce the fact that with Renku plugins can be built. In particular with those, new CLI commands can be developed and customized annotations can be added to the graph.

Those would help me to introduce the two plugins we developed.

Of course leaving out technical details regarding the development of the plugin itself.

What do you think?

Reply:
You can say that there are plugins for renku, one developed specifically for astronomy and another more generally. 

This section can not be called "developing renku plugins" since domain experts do not develop plugins and they hardly care that features can be added to plugins. In principle, features can be added to anything, renku itself included, we collaborated on adding quite some and might do more.

I am also not sure the concept of plugins needs general introduction, it's commonly understood. But, within the section for each plugin, it can be said that the plugin should be installed separately, since it is a plugin. With a command showing how to install and also with a suggestion to how to add requirement in the repo.

Reply:
thanks, yes.

I tried to rewrite it not mentioning anything about the renku plugin development itself, but rather focusing on the added functionalities offered by the two plugins within the context of the development of the workflows

So, I included it as part of the paragraph `Developing service in Renku`

What do you think? Should it have a separate dedicated paragraph? I am asking since I don't really know what this paragraph was intended for in first place